adapters/improvedigital/improvedigital.go

```diff
@@ -144,3 +148,87 @@ func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType,
 		Message: fmt.Sprintf("Failed to find impression for ID: \"%s\"", impID),
 	}
 }
+
+// This method responsible to check/validate/apply additional consent providers
```

Comment:
Actually additional consent provider are already present in the request as a string. This function adds them as an array in user.ext.consented_providers_settings.consented_providers. Please update the comment.